### PR TITLE
[WIP] Add new theme variables

### PR DESCRIPTION
**Pull request by nicolasmartinrojo**

### CODE REVIEW

File: src/components/side-panel/layer-panel/layer-config-group.js [hunk 1]
```diff
@@ -27,20 +27,33 @@ import Switch from 'components/common/switch';
...
+  }
+`;
 export const StyledLayerConfigGroupLabel = styled.div`
-  border-left: 2px solid ${props => props.theme.labelColor};
+  border-left: ${props => props.theme.styledLayerConfigGroupLabelBorderLeft} solid
+    ${props => props.theme.labelColor};
   line-height: 12px;
-  margin-left: -12px;
-  padding-left: 10px;
+  margin-left: ${props => props.theme.styledLayerConfigGroupLabelMargin};
```
Comment (heshan0131) [inline src/components/side-panel/layer-panel/layer-config-group.js:41]: you can just call thses`layerConfigGroupLabelMargin`

File: src/components/side-panel/layer-panel/layer-config-group.js [hunk 2]
```diff
@@ -24,23 +24,35 @@ import styled from 'styled-components';
 import classnames from 'classnames';
 import {FormattedMessage} from 'react-intl';
 import Switch from 'components/common/switch';
-import InfoHelper from 'components/common/info-helper';
+import InfoHelperFactory from 'components/common/info-helper';
 import {VertThreeDots} from 'components/common/icons';
 
+const dashedPrependCss = `
```
Comment (heshan0131) [inline src/components/side-panel/layer-panel/layer-config-group.js:30]: why do you need to add dash here? remove this

File: src/components/side-panel/layer-panel/layer-config-group.js [hunk 3]
```diff
@@ -24,23 +24,35 @@ import styled from 'styled-components';
...
   line-height: 12px;
-  margin-left: -12px;
-  padding-left: 10px;
+  margin-left: ${props => props.theme.layerConfigGroupLabelMargin};
+  padding-left: ${props => props.theme.layerConfigGroupLabelPadding};
+
   display: flex;
   align-items: center;
 
+  ${props => (props.theme.layerConfigGroupLabelDashedPrepend ? dashedPrependCss : '')}
```
Comment (heshan0131) [inline src/components/side-panel/layer-panel/layer-config-group.js:47]: remove

File: src/components/side-panel/layer-panel/layer-configurator.js [hunk 4]
```diff
@@ -46,7 +46,10 @@ const StyledLayerConfigurator = styled.div.attrs({
   className: 'layer-panel__config'
 })`
   position: relative;
-  margin-top: 12px;
+  margin-top: ${props => props.theme.styledLayerConfiguratorMargin};
+  padding-top: ${props => props.theme.styledLayerConfiguratorPadding};
+  // border-left: ${props => props.theme.styledLayerConfiguratorBorder}px dashed
```
Comment (heshan0131) [inline src/components/side-panel/layer-panel/layer-configurator.js:51]: delete comment out code

File: src/components/side-panel/layer-panel/layer-config-group.js (hunk 2 above)
Comment (nicolasmartinrojo) [reply src/components/side-panel/layer-panel/layer-config-group.js]: removed

File: src/components/side-panel/layer-panel/layer-config-group.js (hunk 3 above)
Comment (nicolasmartinrojo) [reply src/components/side-panel/layer-panel/layer-config-group.js]: removed

File: src/components/side-panel/layer-panel/layer-config-group.js (hunk 1 above)
Comment (nicolasmartinrojo) [reply src/components/side-panel/layer-panel/layer-config-group.js]: removed

**Comment by heshan0131** [comment]
you should run `yarn lint` and `yarn test` in kepler.gl root repo

**Pull request closed without merging**